README.md

```diff
@@ -1,19 +1,24 @@
-# userVhost
+# rb-ldap
```

Comment:
Perhaps repo rename would also be suiting then. But that's more of a question to @redbrick (o however you tag CMT now)

Reply:
i did specify a repo rename above

Reply:
Appologies.